src/main/java/graphql/execution/instrumentation/parameters/InstrumentationFieldCompleteParameters.java

```diff
@@ -48,7 +48,13 @@ public ExecutionStepInfo getExecutionStepInfo() {
         return executionStepInfo.get();
     }
 
-    public Object getFetchedValue() {
+    /**
+     * This returns the object that was fetched, ready to be completed as a value.  This can sometimes be a {@link graphql.execution.FetchedValue} object
+     * but most often it's a simple POJO.
+     *
+     * @return the object was fetched, ready to be completed as a value.
+     */
+    public Object getFetchedObject() {
```

Comment:
This is the hard breaking change - the new method means they HAVE to handle it should they be using the old method